api/chat.go

```diff
@@ -23,11 +23,28 @@ var m *melody.Melody
 
 const maxParticipants = 10000
 
+func ChatAccessChecker() gin.HandlerFunc {
+	return func(c *gin.Context) {
+		foundContext, exists := c.Get("TUMLiveContext")
+		if !exists {
+			sentry.CaptureException(errors.New("context should exist but doesn't"))
+			c.AbortWithStatus(http.StatusInternalServerError)
+			return
+		}
+		tumLiveContext := foundContext.(tools.TUMLiveContext)
+		if tumLiveContext.Stream.ChatEnabled && tumLiveContext.Course.ChatEnabled {
```

Comment:
Could you add a test for the `ChatEnabled == false` case?

Reply:
Sure, I will

Reply:
Done